src/category_theory/monad/limits.lean

```diff
@@ -82,6 +83,86 @@ def forget_creates_limits (D : J ⥤ algebra T) [has_limit.{v₁} (D ⋙ forget
       end },
     uniq' := λ s m w, by { ext1, ext1, simpa using congr_arg algebra.hom.f (w j) } } }
 
+namespace forget_creates_colimits
+
+variables (D : J ⥤ algebra T) [has_colimit.{v₁} (D ⋙ forget T)]
+
+@[simps] def γ : ((D ⋙ forget T) ⋙ T) ⟶ (D ⋙ forget T) := { app := λ j, (D.obj j).a }
+
+@[simps]
+def c : cocone ((D ⋙ forget T) ⋙ T) :=
+{ X := colimit (D ⋙ forget T),
+  ι := γ D ≫ (colimit.cocone (D ⋙ forget T)).ι }
+
+variable [preserves_colimits_of_shape J T]
+
+@[reducible, nolint doc_blame]
+def lambda :=
+(preserves_colimit.preserves T (colimit.is_colimit (D ⋙ forget T))).desc (c D)
+
+@[nolint doc_blame]
```

Comment:
```suggestion
```

Lemmas are not required to have docstring, there's no need to disable the linter here.